pkg/plugin/keyvault.go

```diff
@@ -87,44 +121,90 @@ func newKeyVaultClient(
 		return nil, fmt.Errorf("failed to get vault url, error: %+v", err)
 	}
 
+	keyIDHash, err := getKeyIDHash(*vaultURL, keyName, keyVersion)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get key id hash, error: %+w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to get key id hash, error: %w", err)
```

In case of `%+v` the field names are added when printing structs. Does adding `%+` with `w` provide any value?

Reply:
It does not. Ihttps://go.dev/play/p/-rGWo9PzFOh

I removed it.